src/main/java/com/smartbear/postman/PostmanImporter.java

```diff
@@ -461,4 +467,39 @@ private String convertTemplateProperties(String postmanUri) {
             return postmanUri.replaceAll("\\{\\{", "{").replaceAll("\\}\\}", "}");
         }
     }
+
+    private class PostmanImporterWorker implements Worker {
```

Comment:
@omitina, what about ``CollectionImporterWorker``? Shouldn't we use this name instead?

Reply:
@Alexander-Sviridov, I based on the parent class - PostmanImporter

Reply:
OK, let's keep it.